src/transformers/models/whisper/modeling_flax_whisper.py

```diff
@@ -821,11 +850,21 @@ def __init__(
         seed: int = 0,
         dtype: jnp.dtype = jnp.float32,
         _do_init: bool = True,
+        gradient_checkpointing: bool = False,
         **kwargs,
     ):
-        module = self.module_class(config=config, dtype=dtype, **kwargs)
+        self.gradient_checkpointing = gradient_checkpointing
```

Comment:
No need to set this as an attribute in the Flax Pre-Trained model class - it only needs to be set as an attribute in the nn.Module 
```suggestion

```